ansible_image_validation/validation-playbooks/ade_packages_validation.yaml

```diff
@@ -0,0 +1,29 @@
+###
+# This Playbook checks if the packages required by Azure Disk Encryption is installed 
+###
+
+---
+
+- name: Check if the packages required by Azure Disk Encryption is installed
+  command: rpm -q "{{ item }}"
+  loop:
+    - lsscsi
+    - psmisc
+    - lvm2
+    - uuid
+    - at
+    - patch
+    - cryptsetup
+    - cryptsetup-reencrypt
+    - procps-ng
+    - util-linux
+  register: package_check
+  ignore_errors: yes
+
+- name: "Write to error msg if ADE packages are not present"
+  lineinfile:
+    path: "{{err_folder}}/err_msgs.log"
+    line: "packages required for ADE are not present"
+    create: yes
+    state: present
+  when: package_check is not succeeded
```

Comment:
Is this going to work as package_check is going to capture different output in case of package not installed. Eg: `package <pkgname> is not installed`

Reply:
it is actually capturing all outputs, but we need to parse and print individual packages. Good suggestion

Reply:
done